modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -73,7 +73,10 @@ For both cases, if a claim is present, the policy will validate the token value,
 
 == Claim Propagation
 
-After the JWT is parsed, the policy sets the claims in the authentication object in a map under the *claims* property.
+Once the policy parses and validates the JWT, the claims are propagated to downstream policies and flows through the *claims* property in the authentication object.
+To access the claim values you can use the following Dataweave 2.0 expression replacing `<claimName>` with the name of the desired claim to access:
```

Comment:
To access claim values, use the following Dataweave 2.0 expression:
`#[authentication.properties.claims.<claimName>]`,
where <claimName> must be the name of the claim that you want to access.